### PR TITLE
Created adapter for Azure App Insights

### PR DESCRIPTION
**Pull request by jwlawrence**
Not much else to say. This uses the v5 tracking snippet from https://github.com/microsoft/ApplicationInsights-JS/commit/d533df90b65c6ebde8d316df082ecbebe6769f13 after I discovered earlier versions were buggy.

### CODE REVIEW

**Comment by jwlawrence** [comment]
@jherdman You seem to be the most active collaborator on here lately. Would you mind taking a look at this PR, please?

File: addon/metrics-adapters/azure-app-insights.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { assert } from "@ember/debug";
+import BaseAdapter from "./base";
+import removeFromDOM from '../utils/remove-from-dom';
+
+export default BaseAdapter.extend({
```
Comment (jherdman) [inline addon/metrics-adapters/azure-app-insights.js:5]: Would you mind converting this into a native class? The Facebook Pixel adapter is a good example.

**Review by jherdman — CHANGES_REQUESTED** [review]
Small change requested.

File: addon/metrics-adapters/azure-app-insights.js (hunk 1 above)
Comment (jwlawrence) [reply addon/metrics-adapters/azure-app-insights.js]: Sure no prob. I just realized that I was working off the blueprint output so I went ahead and updated the blueprint to produce native classes as well.

**Review by jherdman — APPROVED** [review]

**Pull request merged**